vcx/libvcx/src/utils/libindy/cache.rs

```diff
@@ -0,0 +1,191 @@
+extern crate serde_json;
+
+use utils::libindy::wallet::{add_record, get_record, update_record_value};
+
+static CACHE_TYPE: &str = "cache";
+static REV_REG_CACHE_PREFIX: &str = "rev_reg:";
+
+///
+/// Cache object for rev reg cache
+///
+#[derive(Serialize, Deserialize, Debug, Default, PartialEq)]
+pub struct RevRegCache {
```

Comment:
May be it's better to rename the structure to `RevRegStateCache` ? (and also update prefix constant above)

Reply:
If we would like to append fields to the same structure, how do you think, is it good idea to fetch whole structure each time?